.devcontainer/make_devcontainers.sh

```diff
@@ -126,6 +126,10 @@ fi
 # Get the devcontainer image version and define image tag root
 readonly DEVCONTAINER_VERSION=$(echo "$matrix_json" | jq -r '.devcontainer_version')
 
+# Internal image compiler versions:
+readonly CUDA99_GCC_VERSION=$( echo "$matrix_json" | jq -r '.cuda99_gcc_version')
```

Comment:
FYI @wmaxey -- the cuda99 host compiler versions are now specified in `matrix.yaml` rather than staying pinned to the newest GHA CI compilers. This way we can update either independently without breaking the other.